src/Editor.css

```diff
@@ -3,8 +3,10 @@
 }
 
 .editor__textarea {
-  display: inline;
+  display: inline-block;
```

Comment:
I guess this is a subtle aesthetic thing but I'm not sure if we want `inline-block` do we? Is this better with very long examples?

Reply:
Yeah, the reason I changed it was for long examples - when we have `border-style: inset;` and `display: inline;`, longer inputs look like this:

![Screen Shot 2020-01-10 at 9 12 01 AM](https://user-images.githubusercontent.com/31713081/72101650-13dd6000-338a-11ea-8e6a-6d1d432de31d.png)

Whereas with `display: inline-block;`, they look like this:

![Screen Shot 2020-01-10 at 9 11 48 AM](https://user-images.githubusercontent.com/31713081/72101695-32dbf200-338a-11ea-81b7-599eec5eb145.png)

Reply:
Makes sense :+1:

Reply:
I agree that `inline-block` is better, but fyi the earlier `inline` example could be fixed by increasing the `line-height`.